### PR TITLE
Landing page tweaks

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: app/presenters/migration/index.rb [hunk 1]
```diff
@@ -18,17 +18,17 @@ def content
...
             url: new_migration_path,
             text: t('begin_link.text'),
             text_long: t('begin_link.text_long')
           },
           call_to_action: call_to_action,
           previews: [
-            preview_data(1),
-            preview_data(2),
-            preview_data(3, is_person: true, button_text: t('preview_3.button_text'))
+            preview_data(1, url: 'https://www.europeana.eu/portal/en/record/2022608/FBIB_FBib_07004_073.html'),
```
Comment (rwd) [inline app/presenters/migration/index.rb]: Remove /en from the Portal URLs.

File: app/presenters/migration/index.rb [hunk 2]
```diff
@@ -18,17 +18,17 @@ def content
...
+            preview_data(2, url: 'https://www.europeana.eu/portal/en/record/2021609/objecten_60411_A_B.html'),
+            preview_data(3, is_person: true, button_text: t('preview_3.button_text')),
+            preview_data(4, button_text: t('preview_4.button_text'), url: 'https://historia-europa.ep.eu/sites/all/themes/custom/heh/images/heh-logo-mute.png'),
```
Comment (rwd) [inline app/presenters/migration/index.rb]: Is this URL correct? Why would we want to link to another copy of the same image?

File: app/presenters/migration/index.rb [hunk 3]
```diff
@@ -50,6 +50,7 @@ def include_nav_searchbar
 
     def call_to_action
       t('call_to_action')
+      false
```
Comment (rwd) [inline app/presenters/migration/index.rb]: Can't we just remove the entire method from this class instead?

File: app/presenters/migration/index.rb (hunk 2 above)
Comment (andyjmaclean) [reply app/presenters/migration/index.rb]: No.  I've updated the links

File: app/presenters/migration/index.rb (hunk 3 above)
Comment (andyjmaclean) [reply app/presenters/migration/index.rb]: Yes.  I've removed it

File: app/presenters/migration/index.rb (hunk 1 above)
Comment (andyjmaclean) [reply app/presenters/migration/index.rb]: Removed

File: app/presenters/migration/index.rb [hunk 4]
```diff
@@ -18,24 +18,24 @@ def content
...
             text_long: t('begin_link.text_long')
           },
-          call_to_action: call_to_action,
           previews: [
-            preview_data(1),
-            preview_data(2),
-            preview_data(3, is_person: true, button_text: t('preview_3.button_text'))
+            preview_data(1, url: 'https://www.europeana.eu/portal/record/2022608/FBIB_FBib_07004_073.html'),
+            preview_data(2, url: 'https://www.europeana.eu/portal/record/2021609/objecten_60411_A_B.html'),
+            preview_data(3, is_person: true, button_text: t('preview_3.button_text'), url: 'https://www.europeana.eu/portal/collections/migration?q=&view=grid'),
```
Comment (rwd) [inline app/presenters/migration/index.rb]: Remove `view=grid` from this URL.

Comment (andyjmaclean) [reply app/presenters/migration/index.rb]: Done

**Pull request merged**